contracts/JBTiered721DelegateStore.sol

```diff
@@ -624,14 +627,14 @@ contract JBTiered721DelegateStore is IJBTiered721DelegateStore {
       if (_tierToAdd.initialQuantity == 0) revert NO_QUANTITY();
 
       // Get a reference to the tier ID.
-      uint256 _tierId = _currentMaxTierId + _i + 1;
+      uint256 _tierId = _currentMaxTierIdOf + _i + 1;
 
       // Add the tier with the iterative ID.
       _storedTierOf[msg.sender][_tierId] = JBStored721Tier({
         contributionFloor: uint80(_tierToAdd.contributionFloor),
         lockedUntil: uint48(_tierToAdd.lockedUntil),
-        remainingQuantity: uint48(_tierToAdd.initialQuantity),
-        initialQuantity: uint48(_tierToAdd.initialQuantity),
+        remainingQuantity: uint40(_tierToAdd.initialQuantity),
```

Comment:
We keep 16bits empty?

Reply:
oo. yeah this should actually be included in the other ManualMint PR